creator-node/src/dbManager.js

```diff
@@ -433,6 +433,51 @@ class DBManager {
       (indexCreation) => indexCreation.relname === 'Files_storagePath_idx'
     )?.length
   }
+
+  static async getNonDirLegacyStoragePathsAndCids(cursor, batchSize) {
+    const queryResult = await models.File.findAll({
+      attributes: ['storagePath', 'multihash'],
+      where: {
+        multihash: { [sequelize.Op.gte]: cursor },
+        type: { [sequelize.Op.ne]: 'dir' }, // TODO: We also need to do something to move over the dir rows. Maybe a separate query/function that runs first
+        storagePath: {
+          [sequelize.Op.notLike]: '/file_storage/files/%',
+          [sequelize.Op.like]: '/file_storage/%'
+        }
+      },
+      limit: batchSize
+    })
+    logger.debug(
+      `queryResult for non-dir legacyStoragePaths with cursor ${cursor}: ${JSON.stringify(
+        queryResult || {}
+      )}`
+    )
+    if (isEmpty(queryResult)) return []
+    return queryResult.map((result) => {
+      return { storagePath: result.storagePath, cid: result.multihash }
+    })
+  }
+
+  static async updateLegacyPathDbRows(copiedFilePaths, logger) {
+    const transaction = await models.sequelize.transaction()
+    try {
+      for (const { legacyPath, nonLegacyPath } of copiedFilePaths) {
+        await models.File.update({
+          storagePath: nonLegacyPath,
+          where: {
+            storagePath: {
+              [sequelize.Op.eq]: legacyPath
+            }
```

Comment:
how's this diff from `storagePath: legacyPath`?

Reply:
you're right the `sequelize.Op.eq` is redundant. removed